Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -305,10 +305,12 @@ extension Channel {
             }
     }
 
-    private func _commonHTTPServerPipeline(configurator: @escaping (Channel) -> EventLoopFuture<Void>,
-                                              h2ConnectionChannelConfigurator: ((Channel) -> EventLoopFuture<Void>)?,
-                                              configureHTTP2Pipeline: @escaping (Channel) -> EventLoopFuture<Void>) -> EventLoopFuture<Void> {
-        let h2ChannelConfigurator = { (channel: Channel) -> EventLoopFuture<Void> in
+    private func _commonHTTPServerPipeline(
+        configurator: @escaping (Channel) -> EventLoopFuture<Void>,
+        h2ConnectionChannelConfigurator: ((Channel) -> EventLoopFuture<Void>)?,
+        configureHTTP2Pipeline: @escaping (Channel) -> EventLoopFuture<Void>
+    ) -> EventLoopFuture<Void> {
+            let h2ChannelConfigurator = { (channel: Channel) -> EventLoopFuture<Void> in
```

Comment:
Don't hate me but this line is now over-indented... 
```suggestion
        let h2ChannelConfigurator = { (channel: Channel) -> EventLoopFuture<Void> in
```

Reply:
Fixing minutia is what this is all about

Reply:
Don't hate me but that should be minutiae, minutia is singular.

Reply:
🤦‍♂️ Getting that correct was my only criterion too